praw/models/reddit/modmail.py

```diff
@@ -182,17 +182,33 @@ def highlight(self):
         """
         self._reddit.post(API_PATH["modmail_highlight"].format(id=self.id))
 
-    def mute(self):
+    def mute(self, duration=None):
         """Mute the non-mod user associated with the conversation.
 
+        :param duration: Duration of mute in days. Valid options are 3, 7, or 28.
+            Defaults to 3 days. (default: None)
+
         For example:
 
         .. code-block:: python
 
            reddit.subreddit("redditdev").modmail("2gmz").mute()
 
+        To mute for 7 days:
+
+        .. code-block:: python
+
+           reddit.subreddit("redditdev").modmail("2gmz").mute(7)
+
+
         """
-        self._reddit.request("POST", API_PATH["modmail_mute"].format(id=self.id))
+        if duration:
+            params = {"num_hours": duration * 24}
+        else:
+            params = {}
+        self._reddit.request(
+            "POST", API_PATH["modmail_mute"].format(id=self.id), params=params
+        )
```

Comment:
I see that this method previously used `self._reddit.request()` but unless there's some reason not to, I think it makes sense to instead do

    self._reddit.post(API_PATH["modmail_mute"].format(id=self.id), params=params)

Reply:
For this, I wasn't sure why `self._reddit.request` was chosen vs `self._reddit.post`. It could be that `self._reddit.post` objectifies the response. For this instance, it isn't needed to be objectified.

Reply:
Okay, that's fine.